app/src/main/res/values/strings-preferences.xml

```diff
@@ -73,7 +68,6 @@
     <!-- Navigiation Slider -->
     <string name="pref_title__sub_nav_slider">Navigation Slider</string>
     <string name="pref_desc__sub_nav_slider">Control visibility of entries in the navigation drawer</string>
-    <string name="pref_cat__visibility_nav_items">Item visibility</string>
```

Comment:
String ressources which are **not** marked as `translateable=false` (which are translatable) have to be removed in all languages where it has been translated too. Else building will fail